DigitalLearningSolutions.Web/Views/FindYourCentre/Index.cshtml

```diff
@@ -1,7 +1,10 @@
 ﻿@using DigitalLearningSolutions.Web.Helpers
+@using DigitalLearningSolutions.Web.ViewModels.FindYourCentre
+@model FindYourCentreViewModel
 @{
   ViewData["Title"] = "Find Your Centre";
   var url = ConfigHelper.GetAppConfig()["CurrentSystemBaseUrl"] + "/findyourcentre?nonav=true";
```

Comment:
Super minor, but could we set this url in the viewmodel so we can easily unit test the url construction and move the logic away from the view? :)